javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,9 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var array = [];
+  for (var i = 0; i < values.length; i++) {
+    if (!isNaN(values[i])) {
+      array.push(values[i]);
+    }
+  } return array;
+}
```

Comment:
This function fails to pass `takes [0, -32, "&@A", 64, "99", -128] and returns [0, -32, 64, -128]`